File: Models/Venda.cs
using System.ComponentModel.DataAnnotations;
namespace MidnightCityTheater.Models;

public class Venda
{
    [Key]
    public int IdVenda { get; set; }
    public DateTime Data { get; set; }
    public Cliente? Cliente { get; set; }
    public Ingresso? Ingresso { get; set; }
    public List<Filme>? Filmes { get; set; }
    public Snack? Snack { get; set; }
    public double PrecoTotalVenda { get; set; }
}

File: Views/FrontMidnightCityTheater/src/app/Venda.ts
import { Cliente } from "./Cliente";
import { Filme } from "./Filme";
import { Ingresso } from "./Ingresso";
import { Snack } from "./Snack";

export class Venda {
    idVenda: number = 0;
    data: Date = new Date();
    cliente: Cliente = new Cliente();
    ingresso: Ingresso = new Ingresso();
    filmes: Array<Filme> = [];
    snack: Snack = new Snack();
    precoTotalVenda: number = 0;
}